pkg/kubelet/cri/remote/remote_runtime.go

```diff
@@ -83,11 +83,11 @@ func (r *remoteRuntimeService) Version(apiVersion string) (*runtimeapi.VersionRe
 		Version: apiVersion,
 	})
 	if err != nil {
-		klog.Errorf("Version from runtime service failed: %v", err)
+		klog.ErrorS(err, "Version from runtime service failed")
 		return nil, err
 	}
 
-	klog.V(10).Infof("[RemoteRuntimeService] Version Response (typedVersion=%v)", typedVersion)
+	klog.V(10).InfoS("[RemoteRuntimeService] Version Response", "typedVersion", typedVersion)
```

Comment:
What is the difference in printed `typedVersion` versus `apiVersion`? 
I think we should use key `apiVersion`, as I think this is still API version but verified at runtime. Please test it and let me know if I'm wrong.
```suggestion
	klog.V(10).InfoS("[RemoteRuntimeService] Version Response", "apiVersion", typedVersion)
```